src/modules/ekf2/EKF/covariance.cpp

```diff
@@ -103,14 +103,43 @@ void Ekf::initialiseCovariance()
 
 void Ekf::predictCovariance(const imuSample &imu_delayed)
 {
-	// Use average update interval to reduce accumulated covariance prediction errors due to small single frame dt values
-	const float dt = _dt_ekf_avg;
+	// gyro_bias: clear covariances if inhibited
+	for (unsigned index = 0; index < State::gyro_bias.dof; index++) {
+		if (_gyro_bias_inhibit[index]) {
+			P.uncorrelateCovariance<1>(State::gyro_bias.idx + index);
```

Comment:
Only do this once when transitioning to state inhibited?

Reply:
Makes sense, I'll update that